simple_api/object/actions.py

```diff
@@ -48,6 +60,7 @@ def __init__(self, parameters=None, data=None, return_value=None, exec_fn=None,
     def get_fn(self):
         if self._fn is None:
             self._fn = TemplateFunction(self.exec_fn)
+            self._fn.set_validation_hook(build_validation_fn(self.action_validators, self.field_validators))
```

Comment:
potrebujeme to mit rozdelene? sam neznam odpoved, tak si to rozmysli a muzes mi to rict :)